### ISSUE TITLE
train.py error  

### ISSUE THREAD

**Issue by zdmwang** [Zhongdao/Towards-Realtime-MOT#56]
On line 39 of syncbn.py: ex_w, exs_w = ctx.master_queue.get (). A queue is blocked. How can I solve it? Thanks.

**Comment by Zhongdao** [comment]
You can try to train with the original BN instead. Simply replace syncbn with nn.BatchNorm [here](https://github.com/Zhongdao/Towards-Realtime-MOT/blob/master/models.py#L12)
